.ko.yaml

```diff
@@ -0,0 +1,9 @@
+---
+builds:
+  - id: imgctrl
+    main: ./cmd/imgctrl
+    flags:
+      - -tags
+      - containers_image_openpgp
+    ldflags:
```

Comment:
Do we know if we could do it from an environment variable (`go env`)?

Reply:
For the record, yes we can [use `CGO_LDFLAGS`](https://matrix-org.github.io/go-neb/pkg/C/index.html):
```
When building, the CGO_CFLAGS, CGO_CPPFLAGS, CGO_CXXFLAGS and CGO_LDFLAGS environment variables are added to the flags derived from these directives. Package-specific flags should be set using the directives, not the environment variables, so that builds work in unmodified environments. 
```

Reply:
Going back on this, it's not only CGO builds so moving the `-ldflags` back to `GOFLAGS`.